modules/schematics/src/ng-add/schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "SchematicsNgRxRoot",
```

Comment:
Might be a bit weird, but the other packages are using the following convention:

```suggestion
  "id": "SchematicsNgRxSchematics",
```